MSAL/src/native_auth/public/state_machine/error/MSALNativeAuthError.swift

```diff
@@ -29,10 +29,18 @@ import Foundation
 public class MSALNativeAuthError: NSObject, LocalizedError {
     /// Describes why an error occurred and provides more information about the error.
     public var errorDescription: String? { message }
+    
+    /// Correlation ID used for the request
+    public let correlationId: UUID
+
+    /// Error codes returned along with the error
+    public let errorCodes: [Int]
```

Comment:
As agreed errorUri field will be added as String in the public SDK error

Reply:
Done